addon/utils/id.js

```diff
@@ -22,3 +22,8 @@ export function helpTextId(id) {
   return `${id}HelpText`;
 }
 export const computedHelpTextId = makeComputedIdGenerator(helpTextId);
+
+export function idVariation(id, suffix) {
+  return `${id}${suffix}`;
+}
+export const computedIdVariation = makeComputedIdGenerator(idVariation);
```

Comment:
Now we've got this, how about refactoring the other helpers above to use `idVariation` instead so all of these utils go through the same function?